src/App.jsx

```diff
@@ -29,19 +29,20 @@ export function App() {
 	const { user } = useAuth();
 	const userId = user?.uid;
 	const userEmail = user?.email;
-
+	// console.log('userID', userId);
+	// console.log('email', userEmail)
```

Comment:
I would echo Andrea and Devina's suggestion to remove these commented out console logs to clean up our files a bit!

```suggestion
```